paperqa/agents/__init__.py

```diff
@@ -65,11 +64,7 @@ def is_running_under_cli() -> bool:
 def set_up_rich_handler(install: bool = True) -> RichHandler:
     """Add a RichHandler to the paper-qa "root" logger, and return it."""
     rich_handler = RichHandler(
-        rich_tracebacks=True,
-        markup=True,
-        show_path=False,
-        show_level=False,
-        console=Console(force_terminal=True),
```

Comment:
This `Console(force_terminal=True)` was leading to the progress bar being refreshed every printout, which re-renders it every print (and blowing up the terminal).

It's unclear why this is here, so I am removing this `console=Console(force_terminal=True)` part

Reply:
Didn't you add this?

Reply:
No it was not me